<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCadastroServicos</class>
 <widget class="QDialog" name="JanelaCadastroServicos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serviço</string>
  </property>
  <widget class="QLineEdit" name="txt_retorno">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>447</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QPushButton" name="pb_consultarServico">
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_alterarServico">
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_removerServico">
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pb_inserirServico">
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_limparServico">
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cb_retorno">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>260</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>retorno:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Clientes*:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>175</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Valor*: R$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_valor"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cb_agendarVacina">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>260</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Agendar Vacina</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_clientes">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>* Campo obrigatório</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>551</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="cb_adestramento">
        <property name="text">
         <string>Adestramento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_banho">
        <property name="text">
         <string>Banho</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="cb_castracao">
        <property name="text">
         <string>Castração</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_consulta">
        <property name="text">
         <string>Consulta</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="cb_doacao">
        <property name="text">
         <string>Doação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_hospedagem">
        <property name="text">
         <string>Hospedagem</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QCheckBox" name="cb_lojinha">
        <property name="text">
         <string>Lojinha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_tosa">
        <property name="text">
         <string>Tosa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cbox_servico">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Serviços:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
